src/reqwest.rs

```diff
@@ -51,36 +35,29 @@ mod blocking {
     ///
     /// Synchronous HTTP client.
     ///
-    pub fn http_client(request: HttpRequest) -> Result<HttpResponse, Error<reqwest::Error>> {
+    pub fn http_client(request: HttpRequest) -> Result<HttpResponse, Error> {
         let client = blocking::Client::builder()
             // Following redirects opens the client up to SSRF vulnerabilities.
             .redirect(RedirectPolicy::none())
-            .build()
-            .map_err(Error::Reqwest)?;
+            .build()?;
 
-        #[cfg(feature = "reqwest")]
         let mut request_builder = client
             .request(request.method, request.url.as_str())
             .body(request.body);
 
         for (name, value) in &request.headers {
             request_builder = request_builder.header(name.as_str(), value.as_bytes());
         }
-        let mut response = client
-            .execute(request_builder.build().map_err(Error::Reqwest)?)
-            .map_err(Error::Reqwest)?;
+        let mut response = client.execute(request_builder.build()?)?;
```

Comment:
nice, I should use `#[from]` more

Reply:
Thanks! I've got some preliminary changes lying around to use this pattern more inside this crate, will see to proposing them in a PR.

Reply:
Done in https://github.com/ramosbugs/oauth2-rs/pull/2391